annoying/fields.py

```diff
@@ -65,6 +70,26 @@ class Page(models.Model):
         page.save()
     """
 
+    def __init__(self, serializer=None, deserializer=None, *args, **kwargs):
```

Comment:
Yes, that was exactly the case. I've fixed it. Unfortunately, there is no way to keep the signature for an IDE introspection.